src/Validator/Constraints/MemberIsNotPayerValidator.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Validator\Constraints;
+
+use App\Entity\Member;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class MemberIsNotPayerValidator extends ConstraintValidator
+{
+    public function validate($value, Constraint $constraint): void
```

Comment:
### 🤓 Nitpick: `$value` is a bad name.

Do we have an `$member` here? if so - just rename prop to add a little bit more info )